NuKeeper/Commands/CommandBase.cs

```diff
@@ -51,11 +51,20 @@ protected CommandBase(IConfigureLogLevel logger)
             _logger = logger;
         }
 
+
         // ReSharper disable once UnusedMember.Global
         public async Task<int> OnExecute()
         {
             _logger.SetLogLevel(Verbosity);
             var settings = MakeSettings();
+
+            var validationResult = ValidateSettings(settings);
+            if (!validationResult.IsSuccess)
+            {
+                Console.WriteLine(validationResult.ErrorMessage);
```

Comment:
I'm wondering perhaps this should write to logger Error reporting?

Reply:
Well, we do have it around, so yes.